### PR TITLE
Feature/add identityattribute introduction

### PR DESCRIPTION
**Pull request by vinzlange**

### CODE REVIEW

File: _includes/scenarios/integrate/identityattribute-introduction.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+An [Identity Attribute]({% link _docs_integrate/attribute-values.md %}#identity-attributes) is a specific piece of information associated with an identity (person or entity) in the context of the networked ecosystem. Attribute values can take on different types that define the format, validation rules, and presentation information for the stored data.
+
+Identity attributes are critical for managing and sharing information within the networked ecosystem and can include both simple, atomic attribute value types (e.g., email address, display name, phone number) and more complex types (e.g., street address, personal name). These attributes may also contain other attribute values, depending on their specific requirements.
```
Comment (stnmtz) [inline _includes/scenarios/integrate/identityattribute-introduction.md:3]: - Cardinality of IdentityAttributes (e.g. multiple given names or addresses)
- The semantics behind attributes count.

File: _includes/scenarios/integrate/identityattribute-introduction.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+An [Identity Attribute]({% link _docs_integrate/attribute-values.md %}#identity-attributes) is a specific piece of information associated with an identity (person or entity) in the context of the networked ecosystem. Attribute values can take on different types that define the format, validation rules, and presentation information for the stored data.
```
Comment (britsta) [inline _includes/scenarios/integrate/identityattribute-introduction.md:1]: - To be consistent with other documentation pages, I would write "IdentityAttribute" instead of "Identity Attribute" (also in other places below) and "Identity" instead of "identity"
- I'd link here ("[Identity Attribute]({% link _docs_integrate/attribute-values.md %}#identity-attributes)") to the IdentityAttribute in the Data Model Overview instead and not yet to the IdentityAttribute value type page. I'd replace in the second sentence "Attribute values" with  "[Attribute values]({% link _docs_integrate/attribute-values.md %}#identity-attributes)" then

File: _includes/scenarios/integrate/identityattribute-introduction.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+## Example
+
+For example, an email address is associated with the attribute value type [EMailAddress]({% link _docs_integrate/attribute-values.md %}#emailaddress), which defines the following:
+
+| Name    | Type             | Required | Validation                                                                                |
+| ------- | ---------------- | :------: | ----------------------------------------------------------------------------------------- |
+| `@type` | `"EMailAddress"` |    ✓     |                                                                                           |
+| `value` | `string`         |    ✓     | min. length: 3<br>max. length: 100<br>must match `^[A-Z0-9._%+-]+@[A-Z0-9.-]+.[A-Z]{2,}$` |
+
+- The type of the attribute
```
Comment (britsta) [inline _includes/scenarios/integrate/identityattribute-introduction.md:14]: - To be consistent with other documentation pages, I would write "Attribute" instead of "attribute" (also in other places above and below)
- I would prefer a standardised punctuation for the bullet points, for example ending all bullet points with a full stop

File: _includes/scenarios/integrate/identityattribute-introduction.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+- The data type of the actual value (a string).
+- How it is validated (according to the pattern of an email address and a maximum length).
+- The requirement or optionality of attributes
+
+## Related links
+
+<!-- - [for more information about creating an IdentityAttribute](% link _docs_integrate/create-own-identityattribute.md %) -->
```
Comment (britsta) [inline _includes/scenarios/integrate/identityattribute-introduction.md:21]: Note that https://github.com/nmshd/documentation/pull/164 is merged now and that its name has changed from "Create own IdentityAttribute" to "Create Attribute for yourself", so it is not only about the creation of IdentityAttributes anymore but a corresponding section within this scenario documentation is about the creation of IdentityAttributes

File: _includes/scenarios/integrate/identityattribute-introduction.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+- [alternative relationship attribute]({% link _docs_integrate/attribute-values.md %}#relationship-attributes)
```
Comment (britsta) [inline _includes/scenarios/integrate/identityattribute-introduction.md:23]: - To be consistent with other documentation pages, I would write "RelationshipAttribute" instead of "relationship attribute". 
- I'd also consider linking the RelationshipAttribute from the Data Model Overview in addition to the RelationshipAttribute value page

**Review by Milena-Czierlinski — COMMENTED** [review]
I really like this introduction and think it helps the reader tremendously understanding the following scenario better.

File: _docs_integrate/attribute-introduction.md [hunk 6]
```diff
@@ -21,3 +21,132 @@ required_by:
...
+{: .notice--info}
+
+### LocalAttributes and IdentityAttributes
+
+From a technical perspective, an IdentityAttribute is always stored as the `content` of a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute). Depending on the values of certain properties of the LocalAttribute, a LocalAttribute whose `content` is given by an IdentityAttribute is also referred to as a **RepositoryAttribute**, an **own shared IdentityAttribute** or a **peer shared IdentityAttribute**.
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/62b3b352-a6d7-4826-8693-e8527b0370e4" id="zVGxoOkr.UEQ"></iframe></div>
```
[the remark this answers is not in the document]
Comment (britsta) [reply _docs_integrate/attribute-introduction.md:57]: - I see your point! I can also see how this could be realised for the IdentityAttribute box. What about the RelationshipAttributes box? There, "you" has RelationshipAttributes that belong to both them and the other party, which means that `owner` doesn't quite fit instead. It would be important to me to have a consistent labelling of the parties in the boxes. Any suggestions? (E.g. try to split the RelationshipAttribute diagramm appropriately like proposed by you to me below?)
- As far as I know, the boxes all have the same width (640px). Did you actually mean the different heights, or is there a formatting setting for the width that I didn't notice?

File: _docs_integrate/attribute-introduction.md [hunk 7]
```diff
@@ -21,3 +21,132 @@ required_by:
...
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/9e4f8ba6-90c6-48e3-a9a3-75d07e70c51b" id="l2jxcORp9Vuq"></iframe></div>
```
[the remark this answers is not in the document]
Comment (britsta) [reply _docs_integrate/attribute-introduction.md:27]: Well analysed! In fact, I initially restricted this to the required properties and also intentionally omitted the `createdAt` property, although I was aware of the inconsistency. As you have brought it up again, I would reconsider adding the `createdAt` property.

However, I see the disadvantage of adding the property in that I would then also add it to the other LocalAttributes in the IdentityAttributes and RelationshipAttributes section for consistency reasons, which would mean a lot of redundant information and would also take up space.

How is your opinion on that?

File: _docs_integrate/attribute-introduction.md [hunk 8]
```diff
@@ -21,3 +21,132 @@ required_by:
...
+
+## RelationshipAttributes
+
+A [RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) is used to store data that is relevant in the context of a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) between two Identities. Both Identities involved in the Relationship must agree to its creation.
+In the context of a single Relationship, each RelationshipAttribute has its unique `key` for identification.
```
[the remark this answers is not in the document]
Comment (britsta) [reply _docs_integrate/attribute-introduction.md:74]: I discussed this with a co-worker a month ago. Apparently there should only be one RelationshipAttribute of a `key` per `owner` within a Relationship, which would also make sense. In fact, this has not yet been implemented or validated, respectively. Together with my co-worker, we decided to put this on the back burner for the time being and have noted it as a to-do instead. 

Let's both implement this feature together soon? The sentence here in the documentation could be left as it is, as this would be good practice when creating RelationshipAttributes. What is your intuition on whether only one  RelationshipAttribute of a certain `key` can be used within a Relationship or at most two, if the `owner` is different? Presumably we have to use the second option anyway because of the third party owned RelationshipAttributes, i.e. take into account differences in ownership?

File: _docs_integrate/attribute-introduction.md [hunk 9]
```diff
@@ -21,3 +21,132 @@ required_by:
...
+
+Sometimes a peer shared RelationshipAttribute is referred to as a peer shared ThirdPartyRelationshipAttribute. This is the case if it can be interpreted as a counterpart of an own shared ThirdPartyRelationshipAttribute. Note that the `shareInfo.sourceAttribute` property of a peer shared ThirdPartyRelationshipAttribute is always undefined, as the own shared RelationshipAttribute used as the source can only be available locally to its owner.
+{: .notice--info}
+
+Although the peer is not its `owner`, the underlying RelationshipAttribute of a peer shared RelationshipAttribute can be shared with peers who are not involved in the Relationship in which the RelationshipAttribute exists if its `confidentiality` is not `"private"`.
+The sharing of the RelationshipAttribute by the peer with another peer is technically documented by the creation of a so-called third party owned RelationshipAttribute.
+
+#### Third party owned RelationshipAttributes
```
[the remark this answers is not in the document]
Comment (britsta) [reply _docs_integrate/attribute-introduction.md:118]: Sure, I would like to do that!

File: _docs_integrate/attribute-introduction.md [hunk 10]
```diff
@@ -21,3 +21,132 @@ required_by:
...
+
+#### Simple IdentityAttributes
+
+A simple IdentityAttribute is an IdentityAttribute with an [IdentityAttribute value type]({% link _docs_integrate/attribute-values.md %}#identity-attributes) for which none of its properties can be interpreted as another IdentityAttribute value type. In most cases, the IdentityAttribute value type then only has a single property, which often also has the name `value`. This property stores the actual value of the IdentityAttribute. From a technical point of view, it can be stated that the `id` of a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) whose `content` is given by a simple IdentityAttribute cannot be the `parentId` of another LocalAttribute. In other words, it could be said that simple IdentityAttributes are not composite. Examples of simple IdentityAttributes are IdentityAttributes for which [DisplayName]({% link _docs_integrate/attribute-values.md %}#displayname) or [EMailAddress]({% link _docs_integrate/attribute-values.md %}#emailaddress) is used as the IdentityAttribute value type.
```
[the remark this answers is not in the document]
Comment (britsta) [reply _docs_integrate/attribute-introduction.md:44]: - I'm having trouble finding a suitable alternative, do you have any suggestions? I don't want to write that the property **has** a different IdentityAttribute value type or **corresponds** to a different IdentityAttribute value type, because this is not really the case: For example, you specify the `day` of a [BirthDate](https://enmeshed.eu/integrate/attribute-values#birthdate) as a string and not as a [BirthDay](https://enmeshed.eu/integrate/attribute-values#birthday), although the `day` property can also be understood as a BirthDay (see validation column) and therefore the corresponding RepositoryAttribute is created.
- 👍 
- 👍

File: _docs_integrate/attribute-introduction.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (Milena-Czierlinski) [reply _docs_integrate/attribute-introduction.md:27]: It's fine for me if you omit it. I just wanted to be sure that you did it on purpose.

File: _docs_integrate/attribute-introduction.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (Milena-Czierlinski) [reply _docs_integrate/attribute-introduction.md:74]: It's fine for me if you leave it here as it is and we take care of the implementation soon.

File: _docs_integrate/attribute-introduction.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (Milena-Czierlinski) [reply _docs_integrate/attribute-introduction.md:57]: - I agree, it would be nice to have it consistent with the following diagram(s). I think the best way in fact is to split the diagram for RealtionshipAttributes. Not only does this improve its comprehensibility, but it would also allow us to declare the owner of an Attribute unambiguously.
- I'll re-check the Attribute boxes and possibly contact you directly.

File: _docs_integrate/attribute-introduction.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (britsta) [reply _docs_integrate/attribute-introduction.md:44]: Good point! Will change it accordingly.

File: _docs_integrate/attribute-introduction.md [hunk 11]
```diff
@@ -84,51 +89,55 @@ The [RelationshipAttribute value types]({% link _docs_integrate/attribute-values
...
+#### Own shared and peer shared ThirdPartyRelationshipAttributes
 
-Sometimes a peer shared RelationshipAttribute is referred to as a peer shared ThirdPartyRelationshipAttribute. This is the case if it can be interpreted as a counterpart of an own shared ThirdPartyRelationshipAttribute. Note that the `shareInfo.sourceAttribute` property of a peer shared ThirdPartyRelationshipAttribute is always undefined, as the own shared RelationshipAttribute used as the source can only be available locally to its owner.
-{: .notice--info}
+Note that it is possible to share the underlying RelationshipAttribute of an [own shared RelationshipAttribute]({% link _docs_integrate/attribute-introduction.md %}#own-shared-and-peer-shared-relationshipattributes) with peers who are not involved in the Relationship in which the RelationshipAttribute exists, provided that the `confidentiality` of the RelationshipAttribute is not `"private"`.
+The sharing of a RelationshipAttribute by its `owner` with such a peer in turn leads to the creation of another own shared RelationshipAttribute in the wallet of the `owner`, which contains the Address of this peer within its `shareInfo.peer` property.
```
Comment (Milena-Czierlinski) [inline _docs_integrate/attribute-introduction.md:114]: "The sharing of a RelationshipAttribute by its owner with such a **third party**"

File: _docs_integrate/attribute-introduction.md [hunk 12]
```diff
@@ -84,51 +89,55 @@ The [RelationshipAttribute value types]({% link _docs_integrate/attribute-values
...
+Such an own shared RelationshipAttribute is referred to as an own shared ThirdPartyRelationshipAttribute.
```
Comment (Milena-Czierlinski) [inline _docs_integrate/attribute-introduction.md:115]: As you already introduced all the terms above, I'd directly use the correct one to avoid confusion. Namely:
"The sharing of a RelationshipAttribute by its owner with such a **third party** in turn leads to the creation of **an own shared ThirdPartyRelationshipAttribute** in the wallet of the owner,"

File: _docs_integrate/attribute-introduction.md [hunk 13]
```diff
@@ -84,51 +89,55 @@ The [RelationshipAttribute value types]({% link _docs_integrate/attribute-values
...
+This is the case as long as the own shared RelationshipAttribute used as the source has not been [deleted]({% link _docs_integrate/delete-attributes.md %}#delete-own-shared-attributes).
 
-Although the peer is not its `owner`, the underlying RelationshipAttribute of a peer shared RelationshipAttribute can be shared with peers who are not involved in the Relationship in which the RelationshipAttribute exists if its `confidentiality` is not `"private"`.
-The sharing of the RelationshipAttribute by the peer with another peer is technically documented by the creation of a so-called third party owned RelationshipAttribute.
+<div style="width: 640px; height: 720px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:720px" src="https://lucid.app/documents/embedded/68fbda7f-30c1-40d1-853a-fd94a91988e8" id="oSgEzd9YW_TQ"></iframe></div>
+
+In the wallet of the peer with whom the underlying RelationshipAttribute of the own shared RelationshipAttribute was shared, a corresponding peer shared RelationshipAttribute is created.
+It is also referred to as a peer shared ThirdPartyRelationshipAttribute.
```
Comment (Milena-Czierlinski) [inline _docs_integrate/attribute-introduction.md:122]: As above, I'd rephrase it to:
"In the wallet of the **third party**, with whom the underlying RelationshipAttribute of the own shared RelationshipAttribute was shared, a corresponding peer shared **ThirdPartyRelationshipAttribute** is created.

File: _docs_integrate/attribute-introduction.md [hunk 14]
```diff
@@ -84,51 +89,55 @@ The [RelationshipAttribute value types]({% link _docs_integrate/attribute-values
...
 
-#### Peer shared RelationshipAttributes
+Of course, the `shareInfo` property of the own shared RelationshipAttribute is set. Within the `shareInfo.peer` property, the Address of the peer to whom the RelationshipAttribute's `owner` has established the Relationship is specified.
+If the RelationshipAttribute is initially created for a specific Relationship and does not originate from another RelationshipAttribute, the `shareInfo.sourceAttribute` property is undefined.
+The peer shared RelationshipAttribute is the peer's [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) and forms the counterpart of the own shared RelationshipAttribute. The `id` of a peer shared RelationshipAttribute is always the same as the `id` of the associated own shared RelationshipAttribute. As with the own shared RelationshipAttribute, the `shareInfo` property of a peer shared RelationshipAttribute is of course set.
+Within the `shareInfo.peer` property, the Address of the `owner` of the RelationshipAttribute is specified.
+It should be emphasized that this is characteristic for a peer shared RelationshipAttribute.
```
Comment (Milena-Czierlinski) [inline _docs_integrate/attribute-introduction.md:106]: When I wrote this comment, I was thinking about the difference between peer shared and third party owned RelationshipAttributes. But now I noticed that it is the same for peer shared RelationshipAttributes and peer shared ThirdPartyRelationshipAttributes. With the strict division between them now, this sentence could be confusing as it might suggest a difference between peer shared RelationshipAttributes and peer shared ThirdPartyRelationshipAttributes which would be wrong. So, I'd kindly ask you to remove it again. 🙈

File: _docs_integrate/attribute-introduction.md [hunk 15]
```diff
@@ -84,51 +89,55 @@ The [RelationshipAttribute value types]({% link _docs_integrate/attribute-values
...
+As this own shared ThirdPartyRelationshipAttribute originates from another own shared RelationshipAttribute that exists in the context of a different Relationship, it contains the `id` of the source own shared RelationshipAttribute within its `shareInfo.sourceAttribute` property.
+This is the case as long as the own shared RelationshipAttribute used as the source has not been [deleted]({% link _docs_integrate/delete-attributes.md %}#delete-own-shared-attributes).
 
-Although the peer is not its `owner`, the underlying RelationshipAttribute of a peer shared RelationshipAttribute can be shared with peers who are not involved in the Relationship in which the RelationshipAttribute exists if its `confidentiality` is not `"private"`.
-The sharing of the RelationshipAttribute by the peer with another peer is technically documented by the creation of a so-called third party owned RelationshipAttribute.
+<div style="width: 640px; height: 720px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:720px" src="https://lucid.app/documents/embedded/68fbda7f-30c1-40d1-853a-fd94a91988e8" id="oSgEzd9YW_TQ"></iframe></div>
```
Comment (Milena-Czierlinski) [inline _docs_integrate/attribute-introduction.md:119]: Here you could actually name "Other peer" "Third party" to improve comprehensibility. In the next section this isn't easily possible. So if you want to keep it as it is for consistency, that is fine with me and you can just resolve this comment.

File: _docs_integrate/attribute-introduction.md [hunk 16]
```diff
@@ -84,51 +89,55 @@ The [RelationshipAttribute value types]({% link _docs_integrate/attribute-values
...
+Although the peer is not its `owner`, the underlying [RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) of a [peer shared RelationshipAttribute]({% link _docs_integrate/attribute-introduction.md %}#own-shared-and-peer-shared-relationshipattributes) can be shared with peers who are not involved in the Relationship in which the RelationshipAttribute exists if its `confidentiality` is not `"private"`. When the peer who has established the Relationship with the `owner` in which the RelationshipAttribute exists shares it, this leads to the creation of two certain [LocalAttributes]({% link _docs_integrate/data-model-overview.md %}#localattribute), the so-called third party owned RelationshipAttributes.
+
+<div style="width: 640px; height: 720px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:720px" src="https://lucid.app/documents/embedded/75177324-b871-4516-b68c-84b106682f81" id="vreEz94.RB0c"></iframe></div>
```
Comment (Milena-Czierlinski) [inline _docs_integrate/attribute-introduction.md:130]: I'd probably swap the pictures of owner and peer, i.e. the owner is an organization and the peer an App user. On the one hand, I think it is more common that App users have Relationships to organizations instead of organizations having Relationships to other organizations (although that is possible of course). On the other hand, that wouldn't suggest that organizations share your data with third parties using enmeshed.

File: _docs_integrate/attribute-introduction.md [hunk 17]
```diff
@@ -149,4 +158,4 @@ If an Identity has created an Attribute that is owned by itself and is no longer
 
 ### Delete Attributes
 
-An Identity may have created an Attribute or received an Attribute from a peer that it does not need any longer. In this case, it can [delete the Attribute]({% link _docs_integrate/delete-attributes.md %}). If an Identity has shared an Attribute that is owned by itself with a peer, it can [request the deletion of this Attribute from the peer]({% link _docs_integrate/delete-attributes.md %}#request-the-deletion-of-own-attributes-from-peer) in order to withdraw their permission to use the Attribute.
+An Identity may have created an Attribute for itself or received an Attribute from a peer that it does not need any longer. In both cases, it can [delete the Attribute]({% link _docs_integrate/delete-attributes.md %}). If an Identity has shared an Attribute that is owned by itself with a peer, it can [request the deletion of this Attribute from the peer]({% link _docs_integrate/delete-attributes.md %}#request-the-deletion-of-own-attributes-from-peer) in order to withdraw their permission to use the Attribute. Of course, the associated [own shared Attribute can be deleted]({% link _docs_integrate/delete-attributes.md %}#delete-own-shared-attributes).
```
Comment (Milena-Czierlinski) [inline _docs_integrate/attribute-introduction.md:161]: Of course, the associated own shared Attribute can be deleted**, too.**

**Review by Milena-Czierlinski — COMMENTED** [review]
Regarding the comments concerning ThirdPartyRelationshipAttributes, perhaps wait integrating them until we had our discussion about this topic, so you don't have to re-do the work.

File: _docs_integrate/attribute-introduction.md (hunk 15 above)
Comment (britsta) [reply _docs_integrate/attribute-introduction.md:119]: I am in favor of avoiding the use of the word "third party" because, depending on the perspective, it differs who the third party is (to be more specific, whether the "peer" or the "other peer" is the third party). To avoid confusion, I would therefore only refer to "peers" instead of "third parties" for the time being.

File: _docs_integrate/attribute-introduction.md (hunk 14 above)
Comment (britsta) [reply _docs_integrate/attribute-introduction.md:106]: No problem 👯‍♀️

File: _docs_integrate/attribute-introduction.md [hunk 18]
```diff
@@ -84,51 +89,55 @@ The [RelationshipAttribute value types]({% link _docs_integrate/attribute-values
...
+The peer shared RelationshipAttribute is the peer's [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) and forms the counterpart of the own shared RelationshipAttribute. The `id` of a peer shared RelationshipAttribute is always the same as the `id` of the associated own shared RelationshipAttribute. As with the own shared RelationshipAttribute, the `shareInfo` property of a peer shared RelationshipAttribute is of course set.
+Within the `shareInfo.peer` property, the Address of the `owner` of the RelationshipAttribute is specified.
+It should be emphasized that this is characteristic for a peer shared RelationshipAttribute.
+In any case, the `shareInfo.sourceAttribute` property of a peer shared RelationshipAttribute is undefined.
+This is plausible if the RelationshipAttribute is initially created for a specific Relationship and does not originate from another RelationshipAttribute, but this is also the case if it originates from another.
+This is because the LocalAttribute which is associated with the RelationshipAttribute and which is used as the source for sharing can only be available locally to the RelationshipAttribute's sharer.
```
[the remark this answers is not in the document]
Comment (britsta) [reply _docs_integrate/attribute-introduction.md:109]: Definitely no :D

[the remark this answers is not in the document]
Comment (britsta) [reply _docs_integrate/attribute-introduction.md:109]: So you would prefer to delete the entire paragraph `"This is plausible if the RelationshipAttribute is initially created for a specific Relationship and does not originate from another RelationshipAttribute, but this is also the case if it originates from another.
This is because the LocalAttribute which is associated with the RelationshipAttribute and which is used as the source for sharing can only be available locally to the RelationshipAttribute's sharer."`? Would be okay for me, I just want to make sure that I understand you correctly.

File: _docs_integrate/attribute-introduction.md (hunk 11 above)
Comment (britsta) [reply _docs_integrate/attribute-introduction.md:114]: I am in favor of avoiding the use of the word "third party" because, depending on the perspective, it differs who the third party is. To avoid confusion, I would therefore only refer to "peers" instead of "third parties" for the time being.

File: _docs_integrate/attribute-introduction.md (hunk 12 above)
Comment (britsta) [reply _docs_integrate/attribute-introduction.md:115]: I know what you mean and will take that into consideration. I think it depends on whether we want to treat "own shared ThirdPartyRelationshipAttributes" as a special case of "own shared RelationshipAttributes" or as a separate LocalAttribute class. What is your first opinion on this? We can probably only decide this once we have clarified the functionalities relating to ThirdPartyRelationshipAttributes in more detail.

File: _docs_integrate/attribute-introduction.md (hunk 13 above)
Comment (britsta) [reply _docs_integrate/attribute-introduction.md:122]: I know what you mean and will take that into consideration. I think it depends on whether we want to treat "peer shared ThirdPartyRelationshipAttributes" as a special case of "peer shared RelationshipAttributes" or as a separate LocalAttribute class. What is your first opinion on this? We can probably only decide this once we have clarified the functionalities relating to ThirdPartyRelationshipAttributes in more detail.

File: _docs_integrate/attribute-introduction.md [hunk 19]
```diff
@@ -84,51 +89,55 @@ The [RelationshipAttribute value types]({% link _docs_integrate/attribute-values
...
 
-#### Own shared RelationshipAttributes
+#### Own shared and peer shared RelationshipAttributes
 
 A [RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) can only exist in the context of a Relationship and must therefore be stored locally for both Identities involved in the Relationship.
 Accordingly, the [creation of a RelationshipAttribute]({% link _docs_integrate/create-attributes-for-yourself.md %}#create-a-relationshipattribute) corresponds to the creation of one [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) for its `owner` and one LocalAttribute for the peer with whom the `owner` has established a Relationship.
```
[the remark this answers is not in the document]
Comment (britsta) [reply _docs_integrate/attribute-introduction.md:97]: I have only found this formulation once more, in the Data Model Overview, and adapted it there. Have you noticed any other places in which this may need to be adapted?

[the remark this answers is not in the document]
Comment (Milena-Czierlinski) [reply _docs_integrate/attribute-introduction.md]: Line 97, 102, 128 of this file.

File: _docs_integrate/attribute-introduction.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (Milena-Czierlinski) [reply _docs_integrate/attribute-introduction.md]: Yes. I'd reformulate the sentence "In any case, the `shareInfo.sourceAttribute` property of a peer shared RelationshipAttribute is undefined." In the next section then, you can mention again that the shareInfo.sourceAttribute of a peer shared ThirdPartyRelationshipAttribute is undefined, too, since the peer doesn't own the source Attribute.

File: _docs_integrate/attribute-introduction.md (hunk 12 above)
Comment (Milena-Czierlinski) [reply _docs_integrate/attribute-introduction.md:115]: - 👍 
- Since you already divided the sections own/peer shared RelationshipAttributes and own/peer shared ThirdPartyRelationshipAttributes, I'd use these terms consistently strictly divided here.

**Review by Milena-Czierlinski — APPROVED** [review]
